pkg/downloader/downloader_test.go

```diff
@@ -80,3 +81,39 @@ func TestDownloadRemote(t *testing.T) {
 		assert.Equal(t, StatusUsedCache, r.Status)
 	})
 }
+
+func TestDownloadLocal(t *testing.T) {
+
+	if testing.Short() {
+		t.Skip()
+	}
+
+	const emptyFileDigest = "sha256:e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855"
+
+	t.Run("without digest", func(t *testing.T) {
+		localPath := filepath.Join(t.TempDir(), t.Name())
+		localFile := filepath.Join(t.TempDir(), "empty.iso")
```

Comment:
No need to add `.iso` suffix
```suggestion
		localFile := filepath.Join(t.TempDir(), "some-file")
```